### PR TITLE
Fix/newtonsoft.json 13.0.1

### PR DESCRIPTION
**Pull request by HermesSbicego-Laser**
This PR Updates Newtonsoft.Json to 13.0.1: it updates web.config (binding redirects), packages.config (nuget) and *.csproj files (projects); It can be used instead of merging those 22 PR made by dependabot few weeks ago.

### CODE REVIEW

File: src/Orchard.Azure.Tests/App.config [hunk 1]
```diff
@@ -29,7 +29,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-12.0.0.0" newVersion="12.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-13.0.0.0" newVersion="13.0.0.0" />
```
Comment (sebastienros) [inline src/Orchard.Azure.Tests/App.config:32]: should it be 13.0.1 ?

**Review by sebastienros — COMMENTED** [review]
OK if the binding redirect is correct.

File: src/Orchard.Azure.Tests/App.config (hunk 1 above)
Comment (HermesSbicego-Laser) [reply src/Orchard.Azure.Tests/App.config:32]: 13.0.0.0 is the assembly version
13.0.1 is the nuget version.
So 13.0.0.0 is the good one for the binding redirect.

Comment (sebastienros) [reply src/Orchard.Azure.Tests/App.config:32]: Good to go then

**Comment by MatteoPiovanelli** [comment]
Just double-checking to confirm this is gtg to merge

**Pull request merged**